apps/playground-api/__tests__/modules/PlaygroundModule.test.ts

```diff
@@ -116,6 +116,12 @@ it('should have gov set', async () => {
   })
   await testing.generate(1)
 
+  await testing.container.waitForPriceValid('BTC/USD')
+  await testing.container.waitForPriceValid('ETH/USD')
+  await testing.container.waitForPriceValid('USDT/USD')
+  await testing.container.waitForPriceValid('CU10/USD')
+  await testing.container.waitForPriceValid('CD10/USD')
+  await testing.container.waitForPriceValid('DUSD/USD')
```

Comment:
yea.. there's somewhere invalid price out there
perhaps you can follow this changes... cleaner
https://github.com/DeFiCh/jellyfish/blob/e653cdce5a2e4d5c9b150ad5427357fd99ac9e9a/apps/playground-api/__tests__/modules/PlaygroundModule.test.ts#L119-L126